Scripts/bintray.json.erb

```diff
@@ -0,0 +1,22 @@
+<% version = Time.now.strftime("%Y.%V") # (year).(week), e.g. 2017.15 %>
```

Comment:
As I'm a ruby dev (and all travis images have one already installed) I used erb as template to dynamically generate the json.